### ISSUE TITLE
integration tests broken in non-ci environment

### ISSUE THREAD

**Issue by banteg** [ApeWorX/ape#879]
### What went wrong?

if you have plugins installed, integration tests break since they expect a clean installation. some functional tests also only work in ci environment.

<img width="1510" alt="Screen Shot 2022-07-15 at 03 02 56" src="https://user-images.githubusercontent.com/4562643/179115238-dee73201-1ebe-430b-b008-0f7bd0c8f7dc.png">

### How can it be fixed?

alter the tests to be less sensitive to the environment they are running in.

**Comment by fubuloubu** [comment]
Can probably do a query of certain things through pip to see what issues to ignore and what not, or do broader checks instead. I think asserting the display of certain elements might be overkill

**Comment by unparalleled-js** [comment]
You have to run tests in an environment that does not have other plugins installed.

**Comment by banteg** [comment]
another inconsistency between pre-commit and ci

<img width="1049" alt="Screen Shot 2022-07-19 at 02 23 51" src="https://user-images.githubusercontent.com/4562643/179627344-9ff17d3d-3251-408e-9e36-cf831ac557f5.png">

**Comment by unparalleled-js** [comment]
> I think asserting the display of certain elements might be overkill

I think it is a good integration test for making sure a network related plugin is installed correctly.

**Comment by fubuloubu** [comment]
> > I think asserting the display of certain elements might be overkill
> 
> I think it is a good integration test for making sure a network related plugin is installed correctly.

Yes, but you can do it like `assert "some text" in result.output` instead
